src/etc/inc/interfaces.inc

```diff
@@ -3007,7 +3007,11 @@ function interface_dhcpv6_prepare($interface = 'wan', $wancfg, $linkdownevent =
     $dhcp6cscript .= "*)\n";
     $dhcp6cscript .= "\t;;\n";
     $dhcp6cscript .= "esac\n";
-
+    $dhcp6cscript .= "if [ -z \$PINFO ]; then\n";
+    $dhcp6cscript .= "rm /tmp/{$wanif}_pdinfo\n";
```

Comment:
```suggestion
    $dhcp6cscript .= "rm -f /tmp/{$wanif}_pdinfo\n";
```

suppress error if file not found (we do not care)

Reply:
We do if it's got a pd the last time and hasn't on this call, it might show a PD that no longer exists.

Reply:
OK, re-read that... we are deleting the file.. :)